src/APIFunctions/2DPrinting.js

```diff
@@ -0,0 +1,80 @@
+import axios from 'axios'
+
+/**
+ * Return an array similar to python's range() function
+ * @param {Number} start
+ * @param {Number} end
+ */
+export const range = (start, end) => {
+  const length = end - start
+  return Array.from({ length }, (_, i) => start + i)
+}
+
+/**
+ * Returns an array of numbers from pages
+ * @param {string} pages
+ * @param {Number} maxPages
+ */
+export function parseRange(pages, maxPages) {
+  var result = new Set()
+  var pagesFromCommaSplit = pages.split(',')
+  pagesFromCommaSplit.forEach(element => {
+    const pagesFromDashSplit = element.split('-')
+    const arr = range(
+      Number(pagesFromDashSplit[0]),
+      Number(pagesFromDashSplit[pagesFromDashSplit.length - 1]) + 1
+    )
+    arr.forEach(element => {
+      result.add(element)
+    })
+  })
+  result.delete(0)
+  result.forEach(element => {
+    if (element > maxPages) result.delete(element)
+  })
+  if (result.size === 0) {
+    var arr = new Set(range(1, maxPages + 1))
+    return arr
+  }
+  return result
+}
+
+/**
+ * Print the page
+ * @param {Object} data
+ */
+export function printPage(data) {
+  axios
+    .post('/api/print/submit', data)
+    .then(res => {
+      console.log(res)
+    })
+    .catch(err => {
+      console.log(err)
+    })
+}
+
+/**
+ *
```

Comment:
you need to explain what this function does on this line, also add documentation for params